templates/security/esm.html

```diff
@@ -8,318 +8,555 @@
-        </div>
-      </div>
-      <div class="col-6 p-divider__block">
-        <div class="p-heading-icon--small">
-          <div class="p-heading-icon__header">
-            {{
-              image(
-              url="https://assets.ubuntu.com/v1/fc781c43-Case+study+-+white.svg",
-              alt="",
-              height="32",
-              width="32",
-              hi_def=True,
-              loading="lazy",
-              attrs={"class": "p-heading-icon__img"},
-              ) | safe
-            }}
-            <p class="p-heading--4">Case study</p>
-          </div>
-          <h4>
-            TIM maintains system security and client confidence with ESM
-          </h4>
-          <p>
-            To ensure the ongoing security of Ubuntu 14.04 LTS machines, ESM provided TIM, the world's largest trade recommendations network, the freedom to upgrade within their own timeframe. This approach saved time and money for this finserv organisation.
-          </p>
-          <p><a href="/engage/case-study-acuris" class="p-button">Read more in the case study</a></p>
-        </div>
+<section class="p-strip is-deep u-no-padding--top">
+    <div class="u-fixed-width">
+      <hr />
+      <h2>What's the difference?</h2>
+      <div class="esm-table">
+        <table class="p-table">
+          <thead>
+            <tr>
+              <th width="50%">Security patching<br /> <span style="text-transform: initial; color: #666666;">(Coverage for critical, high and selected medium CVEs)</span></th>
+              <th width="33.33%" class="u-align--left">Ubuntu LTS</th>
+              <th width="33.33%" class="u-align--left">Ubuntu Pro (Infra-only)<br /> <span style="text-transform: initial; color: #666666;">(Previously known as ”Ubuntu Advantage for Infrastructure”)</span></th>
+              <th width="33.33%" class="u-align--left">Ubuntu Pro</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>Over 2,300 packages in Ubuntu Main repository</td>
+              <td class="u-align--left">5 years </td>
+              <td class="u-align--left">10 years</td>
+              <td class="u-align--left">10 years</td>
+            </tr>
+            <tr>
+              <td>Over 23,000 packages in Ubuntu Universe repository</td>
+              <td class="u-align--left">Best effort</td>
+              <td class="u-align--left">Best effort</td>
+              <td class="u-align--left">10 years</td>
+            </tr>
+          </tbody>
+        </table>
       </div>
     </div>
   </section>
-  <section class="p-strip is-bordered">
+    
+  <section class="p-strip is-deep u-no-padding--top">
+    <div class="u-fixed-width">
+      <table class="p-table" style="border-spacing:0;">
+        <tbody>
+          <tr>
+            <td colspan="2" style="border-top: 1px solid #E5E5E5;" width="28%"><strong>2,300</strong> Ubuntu Main packages, plus<br /><strong>23,000</strong> Ubuntu Universe packages</td>
```

Comment:
This is different in the design

Reply:
So the design is the first block on the right, not the implementation one. That one is a copy paste of a previous version of the PR. So this look correct I think, we discussed this in a meeting yesterday

![image](https://user-images.githubusercontent.com/14939793/203557144-82fb31fd-1ce1-41a9-979a-b1db4f548932.png)